src/plugins/WebOfTrust/util/DelayedBackgroundJob.java

```diff
@@ -0,0 +1,26 @@
+package plugins.WebOfTrust.util;
+
+/**
+ * A {@link BackgroundJob} that is executed after triggers have been aggregated for some delay.
+ *
+ * @author bertm
+ */
+public interface DelayedBackgroundJob extends BackgroundJob {
+    /**
+     * Triggers the execution of the job after either aggregating triggers for at most some delay
+     * specified by the implementation or as soon as the currently running job finishes, whichever
+     * comes last. Implementations must ensure that this method is safe to invoke from any thread
+     * at any moment.
+     */
+    @Override
+    public void triggerExecution();
```

Comment:
Or to say things the other way around: If we kept the "at most" way of saying it, then the purpose of this class would sound like it is a scheduler to execute things which are time critical, i.e. which need to happen ASAP, before a specific point in time. It really sounded like it would execute them immediately, and the time limit was to give it a chance to postpone the stuff for only for some short time bounded by an upper limit.